website/docs/expansion.md

```diff
@@ -210,17 +210,16 @@ kind: Assign
 metadata:
   name: add-sidecar
 spec:
-  source: Generated
   applyTo:
-  - groups: [""]
-    kinds: ["Pod"]
-    versions: ["v1"]
+    - groups: [""]
+      kinds: ["Pod"]
+      versions: ["v1"]
   match:
     scope: Namespaced
-    origin: "Generated"
+    source: All
```

Comment:
@davis-haba i couldn't get this to work with `Generated`, any ideas?

Reply:
Sorry, I think this was a mistake in the example. The `source` field doesn't really makes sense on an `ExpansionTemplate`. It is intended for mutators and constraints which seek to only target generated resources. I suppose if we implement recursive expansion, it might make sense to want an `ExpansionTemplate` to only match `Generated` resources, but this seems niche. 

In this example, I think we should just remove the `source` field from the `ExpansionTemplate`.

Reply:
Just to confirm, you were able to get it to work with `source: "All"` but not `source: "Generated"`?

Reply:
I tested on a kind v1.25 cluster. I was expecting to see pod to get injected sidecar and args to be mutated.
Here's a quick recording: https://asciinema.org/a/qEoR30Hl210qL0Tw70ZiOaBwz

Tested with `gator expand` too, works there.

Reply:
Since the mutators have `source: Generated` they will not apply to the real pod that gets created on the cluster. Only the ephemeral expanded pod, that is and used for validating the deployment, will be mutated.

Reply:
That makes sense. I think what's confusing is we say "When expanded, the above configs will produce the following Pod:" which is not true with `source: Generated`.

I'll update with `source: All` if that sounds good to you? @davis-haba

Reply:
Maybe also update `When expanded, the above configs will produce the following Pod` to `ephemeral expanded pod`

Reply:
Yep, I agree that could be confusing. I think changing the example to `source: All` makes sense.

Thanks!